package/MDAnalysis/lib/iterators.pxd

```diff
@@ -0,0 +1,24 @@
+from libcpp.vector cimport vector
+from libc.stdint cimport int64_t
+
+cdef extern from "iterators.h":
+    cdef cppclass _AtomGroupIterator:
+        int64_t n_atoms
+        vector[int64_t] ix
```

Comment:
why can't this (and below) be an `int64_t*`?

Reply:
Good point will change

Reply:
Changed